webapplication/sip/settings.py

```diff
@@ -165,6 +165,7 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+MEDIA_PATH = 'media'
```

Comment:
/media and you'll need to update uwsgi.ini

Reply:
done